### PR TITLE
Adding new libraries needed by the magnet-scanner-ios to give results…

### LINKED ISSUE #261: Add virtual beacons to ios client

### PR DESCRIPTION
**Pull request by arcturus**
… based on geolocation

Fixes #261

### CODE REVIEW

**Comment by arcturus** [comment]
Just one thing to take into account, in the Carthage file now it's pointing to the branch that contains the changes in the `magnet-scanner-ios` framework.

Once the changes land there, this PR will point again to that repo.

**Comment by arcturus** [comment]
@wilsonpage r?

**Comment by arcturus** [comment]
Just to be clear, this adds 'active' geo searching. 

What does mean active?
When you open the app, or perform the refresh action, that's an active research.

File: ios/Cartfile [hunk 1]
```diff
@@ -1,2 +1,4 @@
 github "stephencelis/SQLite.swift" "swift-2.3"
-github "mozilla-magnet/magnet-scanner-ios" ~> 0.2.3
+github "arcturus/magnet-scanner-ios" "geo-scanner"
```
Comment (wilsonpage) [inline ios/Cartfile]: Now this has landed we can change this back?

File: ios/Cartfile [hunk 2]
```diff
@@ -1,2 +1,4 @@
...
+github "Alamofire/Alamofire" ~> 3.5.0
+github "SwiftyJSON/SwiftyJSON" "swift2"
```
Comment (wilsonpage) [inline ios/Cartfile]: These are dependencies of `magnet-scanner-ios` no? Why do they have to be redefined in the host project?

Comment (arcturus) [reply ios/Cartfile]: You are right, we don't need to add the dependencies here, they will be downloaded, but we still need to manually link them in the project.

File: ios/Cartfile.resolved [hunk 3]
```diff
@@ -1,2 +1,4 @@
+github "Alamofire/Alamofire" "3.5.1"
 github "stephencelis/SQLite.swift" "8b95a0c4a884f35e5102cd306f634680255473bb"
-github "mozilla-magnet/magnet-scanner-ios" "0.2.3"
+github "SwiftyJSON/SwiftyJSON" "54017d514a87b2b855b407131292c510cdeb65f8"
+github "mozilla-magnet/magnet-scanner-ios" "0.3.0"
```
Comment (wilsonpage) [inline ios/Cartfile.resolved:4]: Assuming this stuff gets added automatically?

Comment (arcturus) [reply ios/Cartfile.resolved:4]: Yes, that's added automatically when you bootstrap via carthage. It resovles the tag or commit that your project is using.

**Comment by wilsonpage** [comment]
r+ shiiiiipit!

**Pull request merged**